### PR TITLE
Fix all files in the mjk-examples directory

### PR DESCRIPTION
**Pull request by mjkoster**
rename to new convention and update to latest schema patterns

### CODE REVIEW

File: mjk-examples/OCF/odmthing-ocf-washer-new.sdf.json [hunk 1]
```diff
@@ -15,22 +15,22 @@
...
-        "0/odmObject/powerswitch",
-        "0/odmObject/washerstate"
+      "odmRequired": [
+        "#/odmThing/oic.d.washer/odmObject/powerswitch",
+        "#/odmThing/oic.d.washer/odmObject/washerstate"
       ],
       "odmObject": {
         "powerswitch": {
-          "odmRef": "<library>#/odmObject/oic.r.switch.binary"
+          "odmRef": "ocf:/#odmObject/oic.r.switch.binary"
```
Comment (WAvdBeek) [inline mjk-examples/OCF/odmthing-ocf-washer-new.sdf.json]: not sure if we have defined this syntax

File: mjk-examples/OCF/odmthing-ocf-washer-new.sdf.json [hunk 2]
```diff
@@ -15,22 +15,22 @@
...
         },
         "washerstate": {
-            "odmRef": "<library>#/odmObject/oic.r.operational.state"
+            "odmRef": "ocf:/#odmObject/oic.r.operational.state"
         },
         "washermode": {
-            "odmRef": "<library>#/odmObject/oic.r.mode"
+            "odmRef": "ocf:/#odmObject/oic.r.mode"
```
Comment (WAvdBeek) [inline mjk-examples/OCF/odmthing-ocf-washer-new.sdf.json]: same here, not sure if this syntax is supported

**Review by WAvdBeek — APPROVED** [review]
pushing this, will improve the playground, not sure if it will fix all issues though

File: mjk-examples/ZCL/odmdata-zcl-transitiontimedata.sdf.json [hunk 3]
```diff
@@ -10,13 +10,13 @@
...
   "odmData": {
-    "transitiontimedata": {
+    "transitionTimeData": {
       "type": "number", 
       "widthInBits": 16, 
       "minimum": 0, 
       "maximum": 65535, 
       "multipleOf": 1, 
-      "unit": "seconds", 
+      "scaleUnit": "seconds",
```
Comment (akeranen) [inline mjk-examples/ZCL/odmdata-zcl-transitiontimedata.sdf.json]: scaleUnits seems underspecified now. We should clarify what it's really for. And should use same SenML units? This should be "s"?

Comment (mjkoster) [reply mjk-examples/ZCL/odmdata-zcl-transitiontimedata.sdf.json]: It's now "unit": "s"

File: mjk-examples/OCF/odmthing-ocf-washer-new.sdf.json (hunk 2 above)
Comment (mjkoster) [reply mjk-examples/OCF/odmthing-ocf-washer-new.sdf.json]: I just removed this example because we don't have namespace references working yet.

**Review by WAvdBeek — APPROVED** [review]

**Review by akeranen — APPROVED** [review]

**Pull request merged**